core/ArchiveProcessor.php

```diff
@@ -266,8 +266,8 @@ public function aggregateDataTableRecords(
      * as metrics for the current period.
      *
      * @param array|string $columns Array of metric names to aggregate.
-     * @param bool|string|string[] $operationToApply The operation to apply to the metric. Either `'sum'`, `'max'` or `'min'`.
-     *                                               Can also be an array mapping record names to operations.
+     * @param bool|string|string[] $operationsToApply The operation to apply to the metric. Either `'sum'`, `'max'` or `'min'`.
```

Comment:
could it be limited to false?


```suggestion
     * @param string|string[]|false $operationsToApply The operation to apply to the metric. Either `'sum'`, `'max'` or `'min'`.
```